File: nivel-00/descobrir-os-dias-de-n-meses/foo.c
#include <stdio.h>
#include <assert.h>

//
// Função que retorna o número de dias
//
int dias(int n_meses) {
    return n_meses * 30;
}

//
// Seu teste
//
int main (){
    assert(210 == dias(7));
    return 0;
}

File: nivel-00/descobrir-os-dias-de-n-meses/foo.js
var assert = require('assert');

//
// Função que retorna o número de dias
//
var dias = function(n_meses) {
    return n_meses * 30;
}

//
// Seu teste
//
try {
    assert.equal(210, dias(7), "deve retornar a total de dias");
} catch(e) {
    console.log(e);
}